src/routes/docs/tutorials/android/step-4/+page.markdoc

```diff
@@ -0,0 +1,509 @@
+---
+layout: tutorial
+title: Authentication
+description: Learn to build an Android app with no backend code using an Appwrite backend.
+step: 4
+difficulty: beginner
+---
+
+# User Flow {% #user-flow %}
```

Comment:
Anyway to simplify this page? This is way too much, maybe break this up, but I think this is borderline overkill.

Reply:
Thanks for pointing this out. I have revamped the entire authentication user-flow, and have scrapped `RegisterActivity`. Now, there's only one activity for authentication, `AuthActivity`. Please check the new user-flow, and LMK what you think.